prediction/tools/model_warmup/README.md

```diff
@@ -0,0 +1,129 @@
+# Model warmup
+
+From TensorFlow Serving [documentation](https://www.tensorflow.org/tfx/serving/saved_model_warmup):
+
+```
+The TensorFlow runtime has components that are lazily initialized, which can cause high latency for the first request/s sent to a model after it is loaded. This latency can be several orders of magnitude higher than that of a single inference request.
```

Comment:
Please use a block quote instead of code syntax here, so readers won't have to scroll to the side to see the whole thing.

> The TensorFlow runtime has components that are lazily initialized, which can cause high latency for the first request/s sent to a model after it is loaded. This latency can be several orders of magnitude higher than that of a single inference request. To reduce the impact of lazy initialization on request latency, it’s possible to trigger the initialization of the sub-systems and components at model load time by providing a sample set of inference requests along with the SavedModel. This process is known as “warming up” the model.